tools/hangar/src/cli.test.ts

```diff
@@ -138,7 +138,7 @@ test.each(validWingFiles)(
       expect(npx_tfJson).toMatchSnapshot("cdk.tf.json");
 
       // get all files in .wing dir
-      const dotWingFiles = await walk.async(path.join(targetDir, ".wing"), {
+      const dotWingFiles = await walk.sync(path.join(targetDir, ".wing"), {
```

Comment:
🤯  What a clean fix @staycoolcall911

Reply:
Yeah, but it didn't work :)
It's not hangar's fault that our snapshot tests are printing inflight files in a non-deterministic order. That actually happens before, either on our compiler or SDK. I'm trying to look into it, but would love pointers